packages/moonstone/CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 - `moonstone/Slider` to highlight knob when selected
 - `moonstone/Slider` to handle updates to its `value` prop correctly
 - `moonstone/ToggleItem` to accept HTML DOM node tag names as strings for its `component` property
+- `moonstone/Picker` state on onMouseLeave
```

Comment:
Let's write CHANGELOG as how it impacts the user rather than explain the implementation details.

Reply:
- `moonstone/Picker` to clear `pressed` state `onMouseLeave`?